composer.json

```diff
@@ -27,7 +27,7 @@
     "illuminate/queue": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|^6.0|^7.0|^8.0",
     "illuminate/routing": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|^6.0|^7.0|^8.0",
     "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|^6.0|^7.0|^8.0",
-    "openzipkin/zipkin": "~2.0.2",
+    "openzipkin/zipkin": "~2.0.2|3.0.x-dev",
```

Comment:
I don't think we should support a dev release of Zipkin at this stage, once it is released we can do so.

Reply:
I would love to avoid it too, I just hope they'll release v3 soon.

Reply:
Agreed, better keep support for stable releases only.